dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -206,6 +206,7 @@ ModelicaError               =             unexpected error while using Modelica
 //--------------- STATIC_DATA -----------------------------------------
 UnknownBus                  =             bus id '%1%' does not exist in the iidm import
 UnknownCalculatedBus        =             calculated bus associated to '%1%' node '%2%' does not exist in the iidm import
+UnknownCalculatedBusWithBbs =             calculated bus associated to '%1%' bbs '%2%' does not exist in the iidm import
```

Comment:
bbs => bus bar section on the message side

Reply:
Fixed

Reply:
This error doesn't exist any more.
Instead, there is a new key CalculatedBusNotFound in DYNLog.